codespan-reporting/tests/snapshots/term__empty__no_color.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: codespan-reporting/tests/term.rs
+expression: TEST_DATA.emit_no_color(&TEST_CONFIG)
+---
+bug: 
```

Comment:
Not sure about this output - seems rather challenging to do anything useful if there is no title? 😬

Reply:
I think it would be a good idea to require at least a title in each diagnostic.

Reply:
Yeah... unsure how to prevent people from doing this though - I guess people will see that it renders kind of weirdly and avoid doing it?